Bigtable/src/Connection/Grpc.php

```diff
@@ -267,55 +275,170 @@ public function setIamPolicy(array $args)
      */
     public function createTable(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $parent = $this->pluck('parent', $args);
+        $columnFamilies = $this->pluck('columnFamilies', $args);
+        $mapField['column_families'] = $this->mapFieldObject($columnFamilies);
```

Comment:
`$mapField` must be defined before you assign to it.

```php
$mapField = [];
$mapField[/** key **/] = 'value';
```

That said, I'm not sure I understand why you're bothering with `$mapField` at all. Can't you pass `$args` into `tableObject()`, then call `mapFieldObject($this->pluck('columnFamilies', $args))` around line 297?

Example of what I'm thinking:

```php
    /**
     * @param array $args
     */
    public function createTable(array $args)
    {
        $parent = $this->pluck('parent', $args);
        return $this->send([$this->bigtableTableAdminClient, 'createTable'], [
            $parent,
            $this->pluck('tableId', $args),
            $this->tableObject($args),
            $this->addResourcePrefixHeader($args, $parent)
        ]);
    }

    /**
     * @param array $args
     * @return Table
     */
    private function tableObject(array &$args)
    {
        return $this->serializer->decodeMessage(new Table(), [
            'columnFamilies' => $this->mapFieldObject($this->pluck('columnFamilies', $args))
        ]);
    }
```